cdisc_rules_engine/models/rule.py

```diff
@@ -48,27 +48,36 @@ def from_cdisc_metadata(cls, rule_metadata: dict) -> dict:
                 "standards": cls.parse_standards(authorities),
                 "classes": rule_metadata.get("Scope", {}).get("Classes"),
                 "domains": rule_metadata.get("Scope", {}).get("Domains"),
-                "rule_type": rule_metadata.get("Rule_Type"),
+                "rule_type": cls.get_key(rule_metadata, "Rule_Type"),
                 "conditions": cls.parse_conditions(rule_metadata.get("Check")),
                 "actions": cls.parse_actions(rule_metadata.get("Outcome")),
             }
 
             if "Operations" in rule_metadata:
                 executable_rule["operations"] = rule_metadata.get("Operations")
 
-            if "Match_Datasets" in rule_metadata:
+            if "Match_Datasets" or "Match Datasets" in rule_metadata:
                 executable_rule["datasets"] = cls.parse_datasets(
-                    rule_metadata.get("Match_Datasets")
+                    cls.get_key(rule_metadata, "Match_Datasets")
                 )
 
-            if "Output_Variables" in rule_metadata.get("Outcome", {}):
-                executable_rule["output_variables"] = rule_metadata.get("Outcome", {})[
-                    "Output_Variables"
-                ]
+            if "Output_Variables" or "Output Variables" in rule_metadata.get(
+                "Outcome", {}
+            ):
+                outcome = rule_metadata.get("Outcome", {})
+                output_variables = cls.get_key(outcome, "Output_Variables")
+                executable_rule["output_variables"] = output_variables
             return executable_rule
         else:
             return rule_metadata
 
+    @classmethod
+    def get_key(cls, data_dict, key):
```

Comment:
I think instead of this, it would be better to take the rule and replace all property name spaces with underscores *recursively* before it is provided to the model. I am worried that some non-root rule properties are accessed elsewhere throughout the engine.